simcon_project/simcon_project/settings.py

```diff
@@ -148,3 +148,12 @@
 MEDIAFILES_DIRS = [
     os.path.join(BASE_DIR, 'static/media')
 ]
+
+SENDGRID_API_KEY = os.getenv('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')
+EMAIL_HOST = 'smtp.sendgrid.com'
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'apikey'
+EMAIL_HOST_PASSWORD = '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'
```

Comment:
i know its base 64 encoded i dont like it tho.. seeems insecure to give github our password

Reply:
Truuue, didn't even think about it. Should we just have everyone input this manually and remove before commits?? Not sure what the best route around this is